covid_moonshot_ml/datasets/moonshot.py

```diff
@@ -104,3 +124,66 @@ def download_achiral(header, fn_out=None):
         achiral_df.to_csv(fn_out, index=False)
 
     return achiral_df
+
+
+# TODO: Generalize inclusion criteria to something more compact
+def download_molecules(
+    header,
+    smiles_fieldname="suspected_SMILES",
+    fn_out=None,
+    fn_cache=None,
+    **filter_kwargs,
+):
+    """
+    Download all molecules and remove any chiral molecules.
+    Parameters
+    ----------
+    header : dict
+        Header information passed to GET request. Must contain an entry for
+        'X-CDD-token' that gives the user's CDD API token
+    smiles_fieldname : str, default='suspected_SMILES'
+        Field to use to extract SMILES
+    fn_out : str, optional
+        If specified, filename to write CSV to
+    fn_cache : str, optional
+        If specified, file to write unfiltered CSV download to
+    filter_kwargs :
+        Other arguments passed to filter_molecules
+
+    Returns
+    -------
+    pandas.DataFrame
+        DataFrame containing compound information for all achiral molecules
+    """
+    import os
+
+    if fn_cache and os.path.exists(fn_cache):
+        with open(fn_cache, "rt") as infile:
+            content = infile.read()
+    else:
+        ## Download all molecules to start
+        url = VAULT_URL + NONCOVALENT_SMI_SEARCH
```

Comment:
We should use f-strings as possible.
```suggestion
        url = f"{VAULT_URL}{NONCOVALENT_SMI_SEARCH}"
```